### PR TITLE
Update pd-recover.md

### PR DESCRIPTION
**Pull request by nolouch**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: pd-recover.md [hunk 1]
```diff
@@ -67,14 +67,18 @@ ansible -i inventory.ini tikv_servers -m shell -a 'cat {{deploy_dir}}/log/tikv*
 
 在指定 `alloc-id` 时需指定一个比当前最大的 `Alloc ID` 更大的值。可以从中控机使用 `ansible ad-hoc`，也可以直接去服务器上翻日志。
 
-#### 从 PD 日志获取 [info] allocates id（推荐）
+#### 从 PD 监控中获取最大的 alloc id （推荐）
```
Comment (TomShawn) [inline pd-recover.md:70]:
```suggestion
#### 从 PD 监控中获取最大的 alloc id（推荐）
```

File: pd-recover.md [hunk 2]
```diff
@@ -67,14 +67,18 @@ ansible -i inventory.ini tikv_servers -m shell -a 'cat {{deploy_dir}}/log/tikv*
...
 
-使用以下命令，从 PD 日志中获取 [info] allocates id：
+在 PD 监控面板下的 Cluster 栏目下，可以找到 `Current ID allocation`, 代表当前已经分配出去的 id 的最大值。
```
Comment (TomShawn) [inline pd-recover.md:72]: `栏目`是指`面板`吗？
```suggestion
在 PD 监控面板下的 Cluster 栏目下，可以找到 `Current ID allocation`, 代表当前已经分配出去的 ID 的最大值。
```

File: pd-recover.md [hunk 3]
```diff
@@ -133,10 +110,11 @@ ansible-playbook start.yml --tags=pd
 
 ### 使用 pd-recover
 
+其中 max-alloc-id 是从日志中或者监控上找到的一个已经分配出去的最大的 alloc id，为了安全，可以将找出来的 id 加个安全的访问，比如加 1000000.
```
Comment (TomShawn) [inline pd-recover.md:113]: `加个安全访问`为什么加的是一个数字？
```suggestion
其中 `max-alloc-id` 是从日志或者监控中找到的一个已经分配出去的最大的 `alloc id`。出于安全考虑，可以将找出来的 ID 加个安全的访问，比如加 `1000000`。

```

**Review by TomShawn — CHANGES_REQUESTED** [review]

**Comment by TomShawn** [comment]
@nolouch Please also involve a technical review.

**Comment by nolouch** [comment]
PTAL @rleungx

**Comment by TomShawn** [comment]
@nolouch Looks like this PR is duplicated with https://github.com/pingcap/docs-cn/pull/4261.

**Pull request closed without merging**